hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/src/main/java/org/apache/hadoop/yarn/conf/YarnConfiguration.java

```diff
@@ -3920,6 +3920,13 @@ public static boolean isAclEnabled(Configuration conf) {
   public static final String DEFAULT_FEDERATION_REGISTRY_BASE_KEY =
       "yarnfederation/";
 
+  public static final String FEDERATION_STATESTORE_HEARTBEAT_INITIAL_DELAY_SECS =
+      FEDERATION_PREFIX + "state-store.heartbeat.initial-delay-secs";
+
+  // 30 secs
+  public static final int
+      DEFAULT_FEDERATION_STATESTORE_HEARTBEAT_INITIAL_DELAY_SECS = 30;
+
```

Comment:
Make the config support rather than binding it just with Seconds

Reply:
Thanks @ayushtkn for reviewing. Can you please elaborate on this? Are you suggesting to remove `-secs` and `_SECS`?

Reply:
yes, don't bind it with seconds, just make seconds as the default type, in case no unit is specified use Seconds, else whatever unit is specified. conf.getTimeDuration should do it for you.
Check this if it help:
https://github.com/apache/hadoop/blob/e0c8c6eed4b123077f350740731d409f2d554e43/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/ShellBasedUnixGroupsMapping.java#L65-L70